### PR TITLE
Design

### PR DESCRIPTION
**Pull request by LukyVj**
To merge ! :leaves:

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -120,15 +125,143 @@ let Marvel = {
...
+      instantsearch.widgets.currentRefinedValues({
+        container: '#current-refined-values',
+        clearAll: 'after'
+      })
+    );
+  },
+  renderHits(){
+    var renderedHits = {
+      render: function(options) {
+        processHeroProfile();
```
Comment (pixelastic) [inline app/app.js]: I think there might be a better way. Here we are adding new bindings everytime a new set of results are displayed. Maybe we could just bind listeners on the parent container? Not blocking right now, but we might have to improve that for later.

File: app/app.js [hunk 2]
```diff
@@ -120,15 +125,143 @@ let Marvel = {
...
+
+     
+      // Appy them
+      profileName.html(heroName);
+      profileAvatar.attr('src', heroAvatar);
+      profileDescription.html('<p>'+heroDesc+'</p>');
+      profileRealName.html(heroSecretId)
 
+      // Check if he is a hero, a vilain, both, or null
+      if(isHero==true && isVillain==false) {
```
Comment (pixelastic) [inline app/app.js]: `isHero` and `isVillain` are already boolean. You can simply write `if (isHero && isVillain)`.

File: app/app.js (hunk 1 above)
Comment (LukyVj) [inline app/app.js]: That is my biggest problem regarding rebinding sicne I'm using Algolia Instantsearch. 
Never certain of what could be the best way to do this. I'll be glad to see how you would manage it!

File: app/app.js (hunk 2 above)
Comment (LukyVj) [inline app/app.js]: That's actually true, I have to remind it, I always do like this, and I already got this kind of comments ^^! Changing it right now!

Comment (LukyVj) [inline app/app.js]: But then, for detecting `false` values, you would do like this ? `if(!isHero && !isVilain)`?

Comment (pixelastic) [inline app/app.js]: Yep, exactly :)

**Pull request merged**